docs/plan-for-production.md

```diff
@@ -248,10 +248,15 @@ cost, but comes with world class commercial support from Red Hat.
 
 When choosing the production operating system to use with your Swarm clusters,
 you should choose one that closely matches what you have used in development and
-staging environments. Although containers abstract much of the underlying OS,
-some things are mandatory. For example, Docker container networks require Linux
-kernel 3.16 or higher. Operating a 4.x kernel in development and staging and
-then 3.14 in production will certainly cause issues.
+staging environments. Although containers abstract much of the underlying OS, 

```

Comment:
@allencloud Thank you for the contribution.  We don't want to maintain these kinds of version details in our documentation.  They are suitable for release notes or blogs about using a particular release.  So, I would change this to provide guidelines rather than specifics :

```markdown
When choosing the production operating system to use with your Swarm clusters,
choose one that closely matches what you have used in development and staging
environments. Although containers abstract much of the underlying OS,  some
features have configuration requirements. For example, to use Docker container
networking with Docker Engine 1.10 or higher, your host must have a Linux kernel
that is version 3.10 or higher. Refer to the change logs to understand the
requirements for a particular version of Docker Engine or Swarm.
```